btree/btree.c

```diff
@@ -2377,6 +2411,288 @@ void m0_btree_ut_node_add_del_rec(void)
 	M0_LEAVE();
 }
 
+/**
+ * This unit test exercises a few tree operations in both valid and invalid
+ * conditions.
+ */
+static void m0_btree_ut_basic_tree_operations(void)
+{
+	void                   *invalid_addr = (void *)0xbadbadbadbad;
+	struct m0_btree        *btree;
+	struct m0_btree_type    btree_type = {.tt_id = M0_BT_EMAP_EM_MAPPING };
```

Comment:
Tree type can be `M0_BT_UT_KV_OPS` for UT.

Reply:
@NikhilBirgade : I am not sure what difference it makes but I will go with your suggestion.